sources.pri

```diff
@@ -79,6 +79,7 @@ SOURCES += \
     src/diskbrowser/diskbrowserdlg.cpp
 
 HEADERS += \
+    $$PWD/include/diskbrowser/diskbrowser.h \
```

Comment:
Is the $$PWD needed?

Reply:
I may have used a QtCreator wizard to generate the new header file which is why it was only added to the .pri file at the top with the unneeded $$PWF.
Good catch! ... fixed.